middleware/middleware.js

```diff
@@ -0,0 +1,7 @@
+const bcrypt = require('bcrypt');
+
+function hash(password){
+  return bcrypt.hash(password, 8)
+}
```

Comment:
Looks like ESLint is not really happy about this one: :bowtie: 

<img src="https://user-images.githubusercontent.com/1170440/36737038-5b5d0a26-1bda-11e8-9e0b-2c5b7bbe2932.png" alt="image" height="80">

Reply:
Saw it, fixed it =)

Reply:
Btw, in this particular case it would be nice to further explain what this number actually means:
```js
function hash(password) {
  return bcrypt.hash(password, 8 /* rounds */);
}
```